Legacy/src/main/java/com/infomaniak/lib/core/githubTools/GitHubViewModel.kt

```diff
@@ -23,16 +23,17 @@ import androidx.lifecycle.liveData
 import com.infomaniak.lib.core.api.ApiController
 import com.infomaniak.lib.core.networking.HttpClient
 import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.invoke
 import okhttp3.Request
 
 class GitHubViewModel : ViewModel() {
 
-    fun getLastRelease(repo: String): LiveData<GitHubRelease?> = liveData(Dispatchers.IO) {
+    fun getLastRelease(repo: String): LiveData<GitHubRelease?> = liveData(Dispatchers.Default) {
```

Comment:
Isn't it a problem that the http call is done on the default dispatcher now ?

Reply:
Oh, I didn't catch the fact that we're using `execute()` and not `await()` here! Looking into it.